src/modules/core/components/FeedbackWidget/FeedbackWidget.css

```diff
@@ -0,0 +1,16 @@
+.button {
+  padding: 0;
+  position: absolute;
+  right: 34px;
+  bottom: 16px;
+  border: none;
```

Comment:
So the feedback fish widget is kinda garbage in the way it works.

I'm sure you've noticed this but upon first render it renders outside of the screen:

![demo-feedback-widget-outside-screen](https://user-images.githubusercontent.com/1193222/107680901-1a35be00-6ca7-11eb-9b9d-aa3135d1b990.gif)

This is because it uses popper under the hood and that needs a center point reference to align properly to it.

Because of this we can hack around it:
- set the width equal to the feedback fish popup
- align your text to the right
- reduce the right positioning (to better align with the feedback fish popup)
- compensate with padding, the lost right position space

So something like this:
```suggestion
  padding: 0;
  position: absolute;
  right: 25px;
  bottom: 16px;
  border: none;
  width: 320px;
  padding-right: 9px;
  text-align: right;
```

This will still keep the feedback link in your original position, but allow the feeback fish popoup to render in the same place every time:

![demo-feedback-widget-proper](https://user-images.githubusercontent.com/1193222/107681212-726cc000-6ca7-11eb-9674-0da8689fef07.gif)